src/main/scala/com/campudus/tableaux/database/model/tableaux/RowModel.scala

```diff
@@ -515,6 +516,18 @@ class RetrieveRowModel(val connection: DatabaseConnection) extends DatabaseQuery
       case (_: LinkColumn, Some(v)) =>
         Json.fromArrayString(v.toString)
 
+      case (MultiLanguageColumn(_: NumberColumn | _: CurrencyColumn), Some(obj)) =>
+        val casedMap = Json.fromObjectString(obj.toString)
+          .asMap
+          .mapValues(n => Option(n) match {
+            case None => null
+            case Some(v: String) => (Try(v.toInt) orElse Try(v.toDouble)).get.asInstanceOf[java.lang.Object]
```

Comment:
Yep, but I don't see that as a problem because this comes straight from the database. If we don't trust the values from database we should shutdown this project.